modules.d/99base/module-setup.sh

```diff
@@ -105,6 +105,22 @@ install() {
                         *) ;;
                     esac
                 done
+
+                for _dev in "${user_devs[@]}"; do
+
+                    case "$_dev" in
+                        /dev/?*) wait_for_dev "$_dev" 0 ;;
+                        *) ;;
+                    esac
+
+                    _pdev=$(get_persistent_dev "$_dev")
+                    [[ $_dev == "$_pdev" ]] && continue
```

Comment:
Why do you need to do this for both `_dev` and `_pdev`?

Reply:
While I can't see the bug, the logic seems reasonable to me.